apps/chronicles/screens/components/AutocompleteScreen.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from "react";
+import { ScrollView, View } from "react-native";
+import {
+    EDSStyleSheet,
+    useStyles,
+    Typography,
+    Spacer,
+    Autocomplete,
+} from "@equinor/mad-components";
+
+const awesomeAnimals = [
+    "Snow Leopard",
+    "Red Panda",
+    "Platypus",
+    "Narwhal",
+    "Axolotl",
+    "Mantis Shrimp",
+    "Tarsier",
+    "Fennec Fox",
+    "Komodo Dragon",
+    "Okapi",
+    "Aardvark",
+    "Quokka",
+    "Kakapo",
+    "Dik-dik",
+    "Thorny Devil",
+    "Sun Bear",
+    "Gharial",
+    "Toucan",
+    "Mandarinfish",
+    "Proboscis Monkey",
+    "Giant Pangolin",
+    "Flying Fox",
+    "Sloth Bear",
+    "Nudibranch",
+    "Ocelot",
+    "Sugar Glider",
+    "Cassowary",
+    "Lemur",
+    "Peacock Mantis Shrimp",
+    "Tamarin",
+];
+
+export const AutocompleteScreen = () => {
+    const styles = useStyles(themedStyles);
+    const [selectedToggleButtons, setSelectedToggleButtons] = useState<string[]>([]);
+    const handleToggle = (animal: string) => {
+        if (selectedToggleButtons.includes(animal)) {
+            // If the animal is already selected, remove it from the array
+            setSelectedToggleButtons(prevState => prevState.filter(item => item !== animal));
+        } else {
+            // If the animal is not selected, add it to the array
+            setSelectedToggleButtons(prevState => [...prevState, animal]);
+        }
+    };
+    const toggleOptions = awesomeAnimals.map(animal => ({
+        title: animal,
+        active: selectedToggleButtons.includes(animal),
+        iconName: selectedToggleButtons.includes(animal) ? "radiobox-marked" : "radiobox-blank",
+        onPress: () => handleToggle(animal),
+    }));
```

Comment:
To me it's a bit strange that the parent component has to handle the state for which options are toggled. Maybe we can look at this together to see how to improve it